src/components/TextField/TextField.tsx

```diff
@@ -56,6 +56,10 @@ export interface Props {
    * HTML label text
    */
   label?: string;
+  /**
+   * Maximum value allowed for the input, if type is 'number'. When the input value matches this maximum, the plus button becomes disabled.
+   */
+  max?: number;
```

Comment:
#2/2 change besides forwarding the ref in this file is matching the `max` type in `InputField` to prevent TS error